app/controllers/api/teams.php

```diff
@@ -449,8 +449,19 @@
         $events
             ->setParam('teamId', $team->getId())
             ->setParam('membershipId', $membership->getId())
+            ->setPayload($response->output(
+                $membership
+                ->setAttribute('teamName', $team->getAttribute('name'))
+                ->setAttribute('userName', $invitee->getAttribute('name'))
+                ->setAttribute('userEmail', $invitee->getAttribute('email'))
+                ->setAttribute('secret', $secret),
+                Response::MODEL_MEMBERSHIP
+            ))
```

Comment:
This seems repetitive. We do almost the same in the response method.

Reply:
I believe we already do that regarding Account/User response format. Should I try to refactor it as part of this PR?

Reply:
Might be simpler on a separate PR that won't block this one. Where have you seen it?

Reply:
Account response model: https://github.com/appwrite/appwrite/blob/master/src/Appwrite/Utopia/Response/Model/Account.php

Notice it extends User, but removes rules regarding passwords. That way password stuff never gets into account model, even if you return it from the action.

Reply:
This would still leave the maintainer with the responsibility of returning different response for each auth method. Would also create too many response models. I was thinking more something in the direction of a a filter that could be assigned once.